wgpu-hal/src/gles/adapter.rs

```diff
@@ -222,6 +222,25 @@ impl super::Adapter {
             .is_none()
             .then_some(())
             .and_then(|_| Self::parse_version(&version).ok());
+        let web_gl = cfg!(all(target_arch = "wasm32", not(target_os = "emscripten")));
```

Comment:
```suggestion
        let web_gl = cfg!(target_arch = "wasm32");
```
Emscripten is still webgl

Reply:
Ah, I guess it just uses more native APIs?

Reply:
Yeah it just pretends it's EGL/GLES 3